Sources/PayPal/Models/PayPalResult.swift

```diff
@@ -0,0 +1,15 @@
+#if canImport(PayPalCheckout)
+@_implementationOnly import PayPalCheckout
+#endif
+
+import Foundation
+
+/// The result of a PayPal payment flow.
+public struct PayPalResult {
+
+    /// The order ID associated with the transaction.
+    public let orderID: String
+
+    /// The Payer ID (or user id) associated with the transaction.
+    public let payerID: String
+}
```

Comment:
@minhthenguyen what has been decided on the other fields that `ApprovalData` provides? like `Buyer`, `Payee`, etc etc, becasue i added them on Android

Reply:
just my opinion but I think for now we'll just have `orderID` and `payerID`.
@scannillo @jaxdesmarais @jonathajones is there any other param you think we should add here?

Reply:
I don't think anything else is needed currently, imo. We can always add things as we see fit, but I think it'll be better to keep the interface as simple as possible for now and iterate on it as merchants ask for certain elements or we see a need. On the NXO side I think Android today offers more data in the ApprovalData object than we do on the iOS side.

Reply:
yes, we have been asked by merchants on the Android github to include more fields. So maybe here we can diverge? idk, but it feels odd to reduce the info we send to the merchant, when we are going to get those complaints again.

Reply:
I think it's better to reduce the scope of the Android PR. The iOS PR is much smaller than the Android one. At this point a reduction in scope is a positive.

Reply:
+1 - we also confirmed this with David that we want to keep this pretty streamlined and basic right now across iOS and Android and then iterate later on once we get actual merchant feedback. 

Jose, I understand your concern and it is good to keep merchant feedback in mind - but the Northstar SDK right now shouldn't be a "kitchen sink" and offer everything that the NXO SDK already does. We also talked with David & Rahul on this recently. The current goal is to offer what the BT SDK already does, and then go from there after merchant feedback. The BT SDK doesn't give back much more advanced details. And if we need them, then we definitely would need them on iOS and Android.

So in summary, we should go ahead with reducing the Android PR 👍 .

Reply:
@scannillo on it.